crates/stages/src/stages/execution.rs

```diff
@@ -240,6 +240,7 @@ where
             // Execute the block
             let execute_start = Instant::now();
 
+            // TODO(alexey): run consensus post block verification?
```

Comment:
@mattsse unsure if we want this. For consistency I think we should run both pre- and post-execution validations using the standalone functions.

Reply:
never mind, we should, because we run the pre-execution validations in the `Bodies` stage.